tests/Merchant_tests.py

```diff
@@ -1,35 +1,85 @@
 import unittest
-from os.path import dirname, join
+from unittest import mock
+from os.path import dirname, join, abspath
+from os import path
+import json
 from ddt import ddt, unpack, data
 from fattmerchant.Merchant import Merchant
 from fattmerchant.Customer import Customer
+from fattmerchant.utils import compare_json_data
 import logging
 
+
 logger = logging.getLogger(__name__)
+test_path = dirname(abspath(__file__))
+static_data = join(test_path, "static_data")
+
+
+def mocked_requests_put(*args, **kwargs):
+    class MockResponse:
+        def __init__(self, json_data, status_code):
+            self.text = json_data
+            self.content = json_data
+            self.status_code = status_code
+    import pdb
+    pdb.set_trace()
+
+# This method will be used by the mock to replace requests.get
+
+
+def mocked_requests_get(*args, **kwargs):
+    class MockResponse:
+        def __init__(self, json_data, status_code):
+            self.text = json_data
+            self.content = json_data
+            self.status_code = status_code
+
+        def json(self):
+            return self.json_data
```

Comment:
Should this not be `self.context`? `json_data` doesn't seem to be part of the class.